README.md

```diff
@@ -85,6 +125,40 @@ console.log(spectral.run({ spec: 'oas2', target: myOAS }));
 //           light: false } } } ]
 ```
 
+## Example implmentations:
+
+- [Spectral Bot](), a GitHub pull request bot using the [Probot]() framework, built by Taylor Barnett
+
+## FAQs
+
+**I want to lint my OpenAPI Specification documents but don't want to implement Spectral right now.**
+
+No problem! A hosted version of Spectral comes **free** with the Stoplight platform. Sign up for a free account [here]().
```

Comment:
https://next.stoplight.io/join

Reply:
Going to get a different link for this from Scott.